main/repl/timerServices.md

```diff
@@ -93,36 +87,37 @@ These are more complex objects that are only used for the following advanced met
 
 ### `TimerWaker` object
 
-A `TimerWaker` has one method, `wake`, which takes a `Timestamp` argument and returns `void`. 
-The passed `timestamp` is the time when the call to `wake()` is scheduled to occur.
+A `TimerWaker` has one method, `wake()`
+- `wake(timestamp)`
+  - `timestamp` `{ Timestamp }`
+  - Returns `undefined`
```

Comment:
`undefined` is more specific than [void](https://www.typescriptlang.org/docs/handbook/2/functions.html#void). This is more than an editorial change. Is that on purpose?

Reply:
No, and I realized at about the same time that you posted this so it's already fixed.